crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs

```diff
@@ -739,8 +754,10 @@ impl Context {
                     }
                 }
                 // Generate the sorted output variables
-                let out_vars =
-                    self.acir_context.sort(input_vars, bit_size).expect("Could not sort");
+                let out_vars = self
+                    .acir_context
+                    .sort(input_vars, bit_size, self.current_side_effects_enabled_var)
+                    .expect("Could not sort");
```

Comment:
This expect "Could not sort" is not very useful as it does not indicate why we don't expect this to trigger if the compiler is logically sound. Since it was here before this PR, its not blocking